modules/sdk/gradle-plugins-target-platform/README.markdown

```diff
@@ -151,14 +153,14 @@ platform.
 
 ### Target Platform BOMs Dependency
 
-The plugin creates a configuration called `targetPlatformBOMs` with no defaults.
+The plugin creates a configuration called `targetPlatformBoms` with no defaults.
 You can use this dependency to set which BOMs to import to configure your target
 platform.
 
 ```gradle
 dependencies {
-	targetPlatformBOMs group: "com.liferay", name: "com.liferay.ce.portal.bom", version: "7.1.0"
-	targetPlatformBOMs group: "com.liferay", name: "com.liferay.ce.portal.compile.only", version: "7.1.0"
+	targetPlatformBoms group: "com.liferay.portal", name: "release.portal.bom", version: "7.2.0"
```

Comment:
We should add another one for the `release.portal.bom.third.party` since that is what we will add for workspace.